blog/2021-09-17-announcing-stryker-net-1.md

```diff
@@ -0,0 +1,265 @@
+| azure-storage-url             | ❌                                          | azure-storage-url             | baseline.azure-fileshare-url |
+| dashboard-fallback-version    | ❌                                          | dashboard-fallback-version    | baseline.fallback-version    |
+| baseline-storage-location     | ❌                                          | baseline-storage-location     | baseline.provider            |
+| dashboard-compare             | with-baseline                               | dashboard-compare             | baseline                     |
+| git-diff-target               | \--since ...                                | git-diff-target               | since.target                 |
+| azure-storage-sas             | azure-fileshare-sas                         | azure-storage-sas             | ❌                           |
+| files-to-exclude              | ❌                                          | ❌                            | ❌                           |
+| test-runner                   | ❌                                          | ❌                            | ❌                           |
+
+❌ means the option has been removed.
+
+## 🚀 Whats new?
+
+An exciting set of new features has been added with this release! Let's walk trough them all:
+
+### Allow failing tests
+
+It's now allowed to start a mutation test run even with failing tests. Stryker will try to make the best of the situation by marking mutants that are covered by `initially failing tests` as `survived`.
+
+### Mutant filtering
+
+It's now possible to filter mutants at source code level. This gives the most fine grained level of control.
+
+The syntax for the comments is: `Stryker [disable|restore][once][all| mutator list][: reason for disabling]`
+
+`// Stryker disable all` Disables all mutations from that line on.
+
+`// Stryker restore all` re-enables all mutations from that line on.
+
+`// Stryker disable once all` will only disable mutations on the next line.
+
+`// Stryker disable once Arithmetic,Update` will only disable Arithmetic and Update mutations on the next line
+
+Example:
+
+```csharp
+var i = 0;
+var y = 10;
+// Stryker disable all : reason for disable
+i++; // won't be mutated
+y++; // won't be mutated
+// Stryker restore all
+i--; // will be mutated
+// Stryker disable once all
+y--; // won't be mutated
+i++; // will be mutated
+// Stryker disable once Arithmetic
+y++; // will be mutated
+// Stryker disable once Arithmetic,Update
+i--; // won't be mutated
+```
+
+### Ignore mutations
+
+The ignore mutations option has been extended to offer more fine grained control. Before v1.x it was possible to ignore complete mutators. Now it's possible to ignore specific mutations inside these mutators as well.
+
+Example:
+
+```json
+"stryker-config": {
+    "ignore-mutations": [
+        "linq.First",
+        "linq.Sum"
+    ]
+}
+```
+
+> Note: this only works for Linq mutations for now.
+
+All possible mutations:
```

Comment:
I don't think this is really neccesary, they can find this in the docs right? We could link to the docs for this.